Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -882,6 +885,7 @@
         - MaterialBananium10
     - type: MaterialStorageMagnetPickup # Frontier
       range: 0.30
+      magnetEnabled: true
```

Comment:
same here

Reply:
Fixed it by making the default to false.